Makefile

```diff
@@ -13,6 +13,8 @@ TOOLING=$(EMBEDMD_BIN) $(JB_BIN) $(GOJSONTOYAML_BIN) $(JSONNET_BIN) $(JSONNETLIN
 
 JSONNETFMT_ARGS=-n 2 --max-blank-lines 2 --string-style s --comment-style s
 
+KUBE_VERSION?="1.18.0"
```

Comment:
Could this be set to `1.20.0` as this is the lowest version currently supported?

Reply:
Definitely, for some reason I thought it was 1.18. Updated.